### PR TITLE
Security review

### PR DESCRIPTION
**Pull request by edeyoung**
Uses prepare statement to increase efficiency of bulk DB inserts. Test against NPPES data (or abridged set of 100k NPPES points). Locally saw 50% reduction in processing time.

Adds better debug info for failing tests (`+v` instead of just `v`)

Adds more info to chpl querier warn statements.

Adds missing tests.

------

Pull requests into this repository require the following checks to be completed by the submitter and reviewers.

**Submitter:**
- [x] Administrative tasks are complete.
  - This pull request describes why these changes were made.
  - The JIRA ticket links to this PR.
  - The JIRA ticket for this PR is: https://oncprojectracking.healthit.gov/support/browse/LANTERN-158
  - vendor/ directories have been updated
    - New dependencies have been added
    - Local dependencies have been updated
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.
- [x] Code review has been performed.
  - Code diff has been done and checked to ensure only expected code is being committed.
  - A linter for the language has been run.

**Primary Reviewer:**

- [x] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [x] Code review has been performed.
  - Code is maintainable, reusable, efficient, and correct.
  - Code accomplishes the tasks purpose.
  - Code follows style guidance appropriate for the language, including passing linter checks.
  - Code is well commented (which does not mean verbosely commented).
  - vendor/ directories have been updated
    - New dependencies have been added
    - Local dependencies have been updated
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.

**Secondary Reviewer:**

- [x] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [x] Code review has been performed.
  - Code accomplishes the tasks purpose.
- [x] Tests are complete.
  - Tests have been run locally and pass.

**Merger:**

Prior to merging, ensure that all checks have been completed. Delete the branch after the merge is complete.

Create a new branch `<branch name>_gomodupdate`. Run `make update_mods` to update the `go.mod`
and `go.sum` files to point to master. Create a PR. Require one sanity check reviewer.

### CODE REVIEW

File: endpointmanager/pkg/endpointmanager/postgresql/healthitproductstore.go [hunk 1]
```diff
@@ -2,11 +2,19 @@ package postgresql
...
 	"context"
+	"database/sql"
 	"encoding/json"
 
 	"github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/endpointmanager"
 )
 
+// prepared statements are left open to be used throughout the execution of the application
+// TODO: figure out if there's a better way to manage this for bulk calls
+var areHealthITProductStatementsPrepared = false
```
Comment (mayerm94) [inline endpointmanager/pkg/endpointmanager/postgresql/healthitproductstore.go:13]: I think that if you make prepareHealthITProductStatements return the prepared statements then you can just assign the gobal variables their values here and then won't need to use  the areHealthITProductStatementsPrepared function since the prepare statement variables will have their values assigned at package initialization.

File: endpointmanager/pkg/endpointmanager/postgresql/fhirendpointstore.go [hunk 2]
```diff
@@ -2,13 +2,21 @@ package postgresql
...
 	"encoding/json"
 
 	"github.com/lib/pq"
 	"github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/capabilityparser"
 	"github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/endpointmanager"
 )
 
+// prepared statements are left open to be used throughout the execution of the application
+// TODO: figure out if there's a better way to manage this for bulk calls
+var areFHIREndpointStatementsPrepared = false
```
Comment (mayerm94) [inline endpointmanager/pkg/endpointmanager/postgresql/fhirendpointstore.go:15]: Same logic described below about relying on package initialization to assign these variables their values so that `areFHIREndpointStatementsPrepared` will not be needed and `prepareFHIREndpointStatements` will not need to be called all over the place
